Tools/GenerateWebsiteData.ps1

```diff
@@ -16,8 +16,12 @@ $apiMappings = @{
 
 # Generate a single JSON file containing all the documentation for all the
 # cmdlets. Unfortunately we can't split these into multiple files because of
-# the way we generating web pages in Jekyll.
+# the way we generating web pages in Jekyll/angular..
 $cmdletDocObjects = @()
+
+#Allows us to separate out cmdlets by product.
+$cmdletDocObjectsGcs = @{}
```

Comment:
Can't we just use a higher-level hash? e.g.

    @{ "Google Cloud Storage" = .... ; "Google Compute Engine" = .... }

Otherwise this will get really ugly when we have cmdlets for a dozen or more products.